tests/test_server.py

```diff
@@ -209,7 +213,7 @@ def test_bookmark_range_api(client):
     assert rd.get_json() == {
         'bookmarks': {
             '1': {'description': '', 'tags': [], 'title': 'Google', 'url': 'http://google.com'},
-            '2': {'description': '', 'tags': [], 'title': 'Example Domain', 'url': 'http://example.com'}}}
+            '2': {'description': '', 'tags': [], 'title': None, 'url': 'http://example.com'}}}
```

Comment:
…What's the purpose of this change?

Reply:
`test_bookmark_range_api` does not pass on master due to bookmark titles not matching between parametrization and assertion.